shared-scripts.sh

```diff
@@ -269,7 +269,7 @@ function _cleanupE2EResources {
   echo "Running clean up script"
   build_batch_arn=$(aws codebuild batch-get-builds --ids $CODEBUILD_BUILD_ID | jq -r -c '.builds[0].buildBatchArn')
   echo "Cleanup resources for batch build $build_batch_arn"
-  yarn clean-e2e-resources --buildBatchArn $build_batch_arn
+  yarn clean-e2e-resources buildBatchArn $build_batch_arn
```

Comment:
This is to fix the bug that the cleanup job in e2e workflow always cleaning all the stale resources. It should only cleanup the resources generated within the batch